static/js/invite.js

```diff
@@ -184,12 +186,12 @@ exports.initialize = function () {
         $("#streams_to_add :checkbox").prop("checked", false);
     });
 
-    $("#submit-invitation").on("click", () => {
-        const is_generate_invite_link = $("#generate_multiuse_invite_radio").prop("checked");
-        if (is_generate_invite_link) {
-            generate_multiuse_invite();
-        } else {
-            submit_invitation_form();
+    $("#submit-invitation").on("click", handle_submit_button_click_events);
+
+    // Ctrl + Enter key to submit form
+    $("#invite-user").on("keydown", (e) => {
+        if (e.key === "Enter" && e.ctrlKey) {
+            handle_submit_button_click_events();
         }
```

Comment:
Should this have a preventDefault/stopPropagation on it?

Reply:
Yes, I will do that.